ui/src/routes/app/[id]/+page.svelte

```diff
@@ -177,19 +186,36 @@
 		<table class="table caption-bottom">
 			<thead class="">
 				<tr>
-					<th class="px-6 py-3 text-left text-xs font-medium tracking-wider uppercase">Title</th>
-					<th class="px-6 py-3 text-left text-xs font-medium tracking-wider uppercase">Author</th>
-					<th class="px-6 py-3 text-left text-xs font-medium tracking-wider uppercase"
+					{#if showImage.v === true}
```

Comment:
Just make sure that this is tidied up once you've moved the state within this file.